pkgs/applications/misc/pass-secret-service/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, python3 }:
+
+with stdenv.lib;
+with python3.pkgs;
+
+buildPythonApplication rec {
+  pname = "pass_secret_service";
```

Comment:
Using underscores in the package name seems a bit unorthodox, if possible, dashes tend to be used instead.

Reply:
I've thought `pname` should be the same as in PyPI, but then the module/package name inside nixpkgs should use dashes as I'm using now. But please correct me if I've understood incorrectly.

Reply:
Had a look among the python packages and there does indeed seem to be a number of packages with `_` in their name so I'll close this comment since there is precedent for using underscores.

Reply:
pname should follow the attribute name and thus be normalized. The PyPI pname can be different, becaus that needs to correspond to the filename on the PyPI server that is downloaded.

Reply:
Ok, I'll make that change! :+1:

Reply:
Fixed.